apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -147,7 +147,8 @@ defmodule Aecore.Structures.SpendTx do
             Application.get_env(:aecore, :tx_data)[:miner_fee_bytes_per_token]
         end
 
-      tx.data.fee >= Float.floor(tx_size_bytes / bytes_per_token)
+      tx.data.fee >= Float.floor(tx_size_bytes / bytes_per_token) &&
```

Comment:
this still uses byte size fee calculation

Reply:
@HristianHristov what do you think about this?

Reply:
Yes It seems I forgot about this one, will be removed.